File: src/main/Java/boj_step10/boj10101.java
import java.io.*;

public class boj10101 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int a = Integer.parseInt(reader.readLine());
        int b = Integer.parseInt(reader.readLine());
        int c = Integer.parseInt(reader.readLine());
        if (a + b + c != 180) writer.append("Error").close();
        else {
            if(a == b && a == c) writer.append("Equilateral").close();
            else if (a == b || a == c || b == c) writer.append("Isosceles").close();
            else writer.append("Scalene").close();
        }
    }
}

File: src/main/Kotlin/boj_step10/boj10101.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val a = r.readLine().toInt()
    val b = r.readLine().toInt()
    val c = r.readLine().toInt()
    if (a + b + c != 180) w.write("Error")
    else {
        if(a == b && a == c) w.write("Equilateral")
        else if (a == b || a == c || b == c) w.write("Isosceles")
        else w.write("Scalene")
    }
    w.close()
}